docs/content/develop/step-4.md

```diff
@@ -9,6 +9,31 @@ menu = "develop"
 tags = [ "develop" ]
 +++
 
+We've been publishing our PatternFly Elements to the [@patternfly organization](https://www.npmjs.com/org/patternfly) on npm.
 
+To roll a new release, use the following steps:
 
-We've been publishing our PatternFly Elements to the [@patternfly organization](https://www.npmjs.com/org/patternfly) on npm.
+1. Start in the root directory of the patternfly-elements project.
+2. Check out the master branch and pull down the latest: `git checkout master && git pull`.
```

Comment:
should we include some other safeguards here, such as: 
` git reset --hard && git clean -df`

Reply:
Applied this with a little styling tweak so didn't commit as-is.